dockers/docker-platform-monitor/docker_init.j2

```diff
@@ -17,6 +16,13 @@ IS_MODULAR_CHASSIS=0
 # Default use python2 version
 SONIC_PLATFORM_API_PYTHON_VERSION=2
 
+if [ -e /usr/share/sonic/hwsku/pmon_daemon_control.json ];
+then
+    PMON_DAEMON_CONTROL_FILE="/usr/share/sonic/hwsku/pmon_daemon_control.json"
```

Comment:
@keboliu is `hwsku` a standard known path? The HWSKU path is something like below
device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700/hwsku.json
or
device/mellanox/x86_64-mlnx_msn2700-r0/<HWSKU>/hwsku.json

Reply:
@prgeor HWSKU is a folder has been mounted to multiple dockers including PMON, please check the following code
https://github.com/sonic-net/sonic-buildimage/blob/master/files/build_templates/docker_image_ctl.j2#L633 

it's a folder which is identical to the path like "device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700/" on the host side.